### DISCUSSION TITLE
Support of OS version 2.14.x

### DISCUSSION THREAD

**Discussion by p10trk** [toltec-dev/toltec#596] [Support] [answered]
As a new user of reMarkable, I am considering trying to adapt Toltec to OS version 2.14.x (whereas the latest supported version is 2.12.3.606). Has anybody attempted this and can share whether there are any major API changes that require deep understanding of the OS and Toltec?

**Comment by Eeems** [comment, marked as answer]
We are waiting for rm2fb and ddvk-hacks to be updated and packages updated in toltec to update our supported version. There is also the standard cycle that happens with packages: https://github.com/toltec-dev/toltec/blob/stable/docs/branches.md

**Comment by palfrey** [comment]
As I noted in https://github.com/toltec-dev/toltec/issues/626, both rm2fb and ddvk-hacks are updated now. Is there any other specific packages that need updating?

**Reply by Eeems** [reply]
Display and ddvk-hacks.

**Comment by palfrey** [comment]
Now solved as of https://github.com/toltec-dev/toltec/pull/635
